meta-resin-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -78,6 +78,12 @@ sync -f "$SYSROOT"
 mv "$SYSROOT/counter.new" "$SYSROOT/counter"
 sync -f "$SYSROOT"
 
+# Run any defined hooks
+for hook in "/etc/hostapp-update-hooks.d/"*; do
+	[ -e "$hook" ] || break
+	"$hook" "$CONTAINER_ID"
```

Comment:
This is a little unclear to me. Should this be "$CONTAINER_ID" or the actual pathname, such as "$SYSROOT/current" or  "$SYSROOT/hostapps/$CONTAINER_ID"? Just using CONTAINER_ID introduces some awkward logic in the upstream user's hooks which should not know about docker containers, they should be just aware of some paths in my opinion.

Reply:
The container id is the target of the update and a hook might need to use the container itself to copy some files etc. Hooks aren't constrained to only deal with what was left in `"$SYSROOT/hostapps/$CONTAINER_ID"`. I think this will become clearer once we start writing the actual hooks